airbyte-integrations/connectors/source-amazon-seller-partner/source_amazon_seller_partner/schemas/ListFinancialEvents.json

```diff
@@ -0,0 +1,89 @@
+{
```

Comment:
With this schema in place, after normalization on Snowflake destination, 1 table was created with property names as `column`.
<img width="1372" alt="Screenshot 2022-05-30 at 1 22 04 PM" src="https://user-images.githubusercontent.com/4735414/171345897-36415838-d9b9-4783-888d-5409897720ad.png">

This gives a good few of fetching the data as per the event list and using them.

When tried with the attached full schema, normalisation was creating 360 tables with each object being tabled separately.

[ListFinancialEvents-with-complete-schema.txt](https://github.com/airbytehq/airbyte/files/8811762/ListFinancialEvents-with-complete-schema.txt)

This may be useful, but more complex tables and representation.

Reply:
Schema has been expanded.